types/entity_tag_test.go

```diff
@@ -0,0 +1,46 @@
+package types_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/cedar-policy/cedar-go"
+	"github.com/cedar-policy/cedar-go/internal/testutil"
+	"github.com/cedar-policy/cedar-go/types"
+)
+
+func TestEntityTagMarshalJSON(t *testing.T) {
```

Comment:
I would put this in `types.entity_test.go`. I don't see a reason for it to live in its own file, really.

Reply:
Yes, done here: https://github.com/cedar-policy/cedar-go/pull/77/commits/9074d5b00f44be4346c13ac8b7270a09b06c541a.